sdk/lib/ucrt/math/_dclass.c

```diff
@@ -11,7 +11,7 @@
 #include <math.h>
 #include <stdint.h>
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && (_MSC_VER >= 1920)
```

Comment:
It looks like
```suggestion
#if defined(_MSC_VER) && (_MSC_VER >= 1922)
```
(Everywhere.)

Reply:
(a reference list for MSVC versions with their corresponding MSC_VER numbers: https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B )

Reply:
If Serge's suggestion is correct, it would have to be brought to all the other changes in this PR.

Reply:
I tested this on https://godbolt.org and agree that this should test should be for 1922 instead of 1920.
PR now changes to reflect this change for all seven (7) locations. Thanks, @SergeGautherie.